bad.rb

```diff
@@ -0,0 +1,7 @@
+class Bad
+  def asdf ds,dfdsf
```

Comment:
Unused method argument - `ds`. If it's necessary, use `_` or `_ds` as an argument name to indicate that it won't be used. You can also write as `asdf(*)` if you want the method to accept any arguments but don't care about them.